pandas/core/frame.py

```diff
@@ -1771,6 +1771,24 @@ def to_numpy(
 
         return result
 
+    def _create_data_for_split_and_tight_to_dict(
```

Comment:
can you type the input & output args here.

Reply:
Thanks @RogerThomas - indeed I missed this. Just missing the return type; I don't think we can say anything more specific than `list` here. Does that give you any mypy issues?